### PR TITLE
PLAT-64: Update ReadMe

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-64: Update ReadMe](https://issues.openmrs.org/browse/PLAT-64)

## Description of what I changed

- Updated ReadMe.md file

## Issue I worked on
[PLAT-64](https://issues.openmrs.org/browse/PLAT-64)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by alexmochu — APPROVED** [review]
LGTM

**Review by dotNesh — APPROVED** [review]
LGTM

**Review by Daramola98 — APPROVED** [review]
LGTM

**Review by emmabaye — APPROVED** [review]
LGTM

**Review by muhozi — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+To run tests 
+ $ npm run test
+ 
+To run coverage
+ $ npm run coverage
+```
+
+## How to contribute to this project?
+ - Please read [OpenMRS wiki](https://wiki.openmrs.org/) for awareness on the code of conduct used in OpenMRS organization 
+ - And to get you started with the project please use the OpenMRS-OCL [documentation](https://docs.google.com/document/d/1R_Fgr5SBl4xFNJgj6yMJNVY63b5H_OUqM55o1GFqFKs/edit#heading=h.rc908wooykzg)
```
Comment (dkayiwa) [inline README.md]: What has the OCL documentation got to do with this?

Comment (dondrzzy) [reply README.md]: Absolutely nothing. I have addressed it.

**Pull request merged**